tests/testsuite/main.rs

```diff
@@ -132,6 +132,14 @@ mod weak_dep_features;
 mod workspaces;
 mod yank;
 
+macro_rules! test_root {
+    () => {
+        std::path::Path::new(file!()).parent().unwrap()
+    };
+}
+
+pub(crate) use test_root;
```

Comment:
1. This might be good to have in `cargo_test_support`
2. Thinking about the name, I think its a bit misleading.  I think people will assume this will be the crate root or `tests/` root.

Neither of these is worth holding this up and the naming issue, I don't have better ideas for.  Still wanted to get this out there to seed thoughts

Reply:
Looks like I have to redo a commit either way so I can change it. Do you like `test_path`?

Reply:
That implies the path to the current file.  What about `test_dir`?  `_root` implies the parent of all dirs while `_dir` might not be so bad.

Reply:
I changed it to `curr_dir` as it seemed to fit the best